### ISSUE TITLE
Build an API on it ?

### ISSUE THREAD

**Issue by Elegye** [SimonDuperray/MessengerSentimentAnalyzer#1]
Will you consider one day to build an API so everyone can analyze their feelings through months and years thanks to your work ?

I will be interested to support this work ...

Alexandre

**Comment by SimonDuperray** [comment]
Yes of course ! I'm working on a new version of the algoritm that will offers new features like a faster data processing, a larger field of feelings and a feature that will can take into account data from everybody, not juste mine.

I was wondering myself about the shape that these new features could come alive. It could be an UI, or just functions that the user could import and use...

What do you think about that ?

Thanks you very much for your interest in this project

Simon
________________________________
De : Alexandre Halopé ***@***.***>
Envoyé : mercredi 7 juillet 2021 15:16
À : SimonDuperray/MessengerSentimentAnalyzer ***@***.***>
Cc : Subscribed ***@***.***>
Objet : [SimonDuperray/MessengerSentimentAnalyzer] Build an API on it ? (#1)


Will you consider one day to build an API so everyone can analyze their feelings through months and years thanks to your work ?

I will be interested to support this work ...

Alexandre

**Comment by Elegye** [comment]
The easiest way to start is to allow user to send its zip file so the user has minimal thing to do to use the library.

In fact, I think it would be awesome to correlate our feelings and our places so we can see where the happier we are.

I think proper Python library is pretty cool so many others can use this work. Actually, [converse](https://pypi.org/project/converse/) offers a similar approach, but it is largely restrained.

A proper online UI on Heroku would be awesome too, with Python backend (as minimal as possible), and React in front of it.
